constrain/library/hpwh_sizing.py

```diff
@@ -0,0 +1,52 @@
+from constrain.checklib import RuleCheckBase
+import numpy as np
+
+PERCENT_TOLERANCE = 0.01  # %
```

Comment:
When I referred to a tolerance I was thinking through a parameter so users could adjust it, for example like in this verification: [            "tol_o": "Tolerance for the number of occupants",](https://github.com/pnnl/ConStrain/blob/a43c892f54a7370bab4319f34888531d51935b31/schema/library.json#L311)

Reply:
Good point. Addressed.